src/pkg/plug/scanner/api/controller.go

```diff
@@ -0,0 +1,157 @@
+// Copyright Project Harbor Authors
```

Comment:
what's `plug` in package path stands for? it doesn't make much sense as the code here is not per plugin, it's the unified shared code.  

Additionally,  there has been `/pkg/scan/` we probably should consider put all scan-related stuff under one sub-dir

Reply:
I'm ok. I'll move the packages to`/scan` and unify the related definitions if necessary.

Reply:
Further package path refactoring will be performed after code completion.